static/app/stores/committerStore.tsx

```diff
@@ -88,7 +90,7 @@ export const storeConfig: Reflux.StoreDefinition & CommitterStoreInterface = {
       },
     };
 
-    this.trigger(this.state);
+    // this.trigger(this.state);
```

Comment:
why are we commenting out this code? Can we leave a comment explaining why?

Reply:
I'll add this back, the PR is still failing iirc

Reply:
Yes, I need to fix the tests to RTL and without changing some things, there's only so much I can do :D

Reply:
I'm fixing tests, I still have a timeout somewhere that's causing an error